modules/graph/loader/arrow_fragment_loader.h

```diff
@@ -158,27 +121,29 @@ class ArrowFragmentLoader {
       std::vector<std::shared_ptr<arrow::Table>> const& partial_v_tables,
       std::vector<std::vector<std::shared_ptr<arrow::Table>>> const&
           partial_e_tables,
-      bool directed = true)
+      bool directed = true, bool generate_eid = false)
       : client_(client),
         comm_spec_(comm_spec),
         vertex_label_num_(vertex_label_num),
         edge_label_num_(edge_label_num),
         partial_v_tables_(partial_v_tables),
         partial_e_tables_(partial_e_tables),
         directed_(directed),
+        generate_eid_(generate_eid),
         basic_arrow_fragment_loader_(comm_spec) {}
 
   ArrowFragmentLoader(vineyard::Client& client,
                       const grape::CommSpec& comm_spec,
                       const std::vector<std::string>& efiles,
-                      bool directed = true)
+                      bool directed = true, bool generate_eid = false)
```

Comment:
Just a little concern about the default value, should it be false? Otherwise all graphs that been loaded into vineyard requires an extra explict argument `generate_eid=True` to make it available for maxgraph. In graphscope, we assume every propery graph could work with gremlin queries.